docs/content/manual/_incoming/2722_VMs_can_not_start_if_a_node_contains_more_than_60_VMs.md

```diff
@@ -0,0 +1,27 @@
+---
+title: VMs can't start if a node contains more than ~60 VMs
+category: console
+tags: VM, p1, acceptance
+---
+Ref: https://github.com/harvester/harvester/issues/2722
+
+![image](https://user-images.githubusercontent.com/5169694/192251104-7a53a1a9-260d-4e90-aade-1b3e7c11cc52.png)
+
+
+### Verify Steps:
+1. Install Harvester with any nodes
+1. Login to console, execute `sysctl -a | grep aio`, the value of `fs.aio-max-nr` should be `1048576`
+1. Update the value by executing:
+```bash
+mkdir -p /usr/local/lib/sysctl.d/
+cat > /usr/local/lib/sysctl.d/harvester.conf <<EOF
+fs.aio-max-nr = 61440
+EOF
+sysctl --system
+```
+1. Execute `sysctl -a | grep aio`, the value of `fs.aio-max-nr` should be `61440`
+1. Reboot the node then execute `sysctl -a | grep aio`, the value of `fs.aio-max-nr` should still be `61440`
+1. Create an image for VM creation
+1. Create 60 VMs and schedule on the node which updated `fs.aio-max-nr`
```

Comment:
Is there any script that's available to provision 60 vms?  Or are we to create 60 vms manually?
It could be great if there was an automated way with building out a giant yaml of CRDs and just applying them to the cluster of kind VirtualMachine and others.  Creating 60vms manually sounds like that might take some time :sweat_smile:.  We could just create a backlog item to create a script to help us with that - or just build an automated test that would preform this.

Reply:
> Creating 60vms manually sounds like that might take some time

when creating VM, select `Multiple instance` could create maximum 10 VMs in the same configuration.

Reply:
Ahh, yes - that's a good point, then we would just need to do that 6 times over to have 60, provisioning 10 at a time.
It still might be worth to build a gist or some yaml later :smile: